documentation/releases.md

```diff
@@ -7,7 +7,7 @@
 
 | Version | Original Release Date | Latest Patch Version | Patch Release Date | End of Support | Runtime Frameworks |
 | --- | --- | --- | --- | --- | --- |
-| 8.0 | October 10, 2023 | [8.0.0](https://github.com/dotnet/dotnet-monitor/releases/tag/v8.0.0) | November 14, 2023 |  | net8.0 |
```

Comment:
/todo [releases] Clear the original release date when promoting a preview version to rtm